sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/OfferEvaluator.java

```diff
@@ -339,13 +336,13 @@ static void logOutcome(StringBuilder stringBuilder, EvaluationOutcome outcome, S
 
         for (VolumeSpec volumeSpec : podInstanceRequirement.getPodInstance().getPod().getVolumes()) {
             evaluationStages.add(VolumeEvaluationStage.getNew(
-                    volumeSpec, Optional.empty(), resourceNamespace, useDefaultExecutor));
+                    volumeSpec, Optional.empty(), resourceNamespace, true));
         }
 
         String preReservedRole = null;
         String role = null;
         String principal = null;
-        boolean shouldAddExecutorResources = useDefaultExecutor;
+        boolean shouldAddExecutorResources = true;
```

Comment:
This boolean can go away

Reply:
this actually seems needed otherwise resources are incorrectly allocated to the default scheduler on every iteration of the below loop